buildscripts/incremental/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source activate travisci
+# Ensure that the documentation builds without warnings
+cd $TRAVIS_BUILD_DIR/docs
```

Comment:
good question. I have no idea, but will find out as I run the script interactively.